### DISCUSSION TITLE
Opinions on sensible battery settings

### DISCUSSION THREAD

**Discussion by HotteX** [Louisvdw/dbus-serialbattery#580] [Q&A] [answered]
Hello Forum
my system works but I could use some help & corrections with fine tuning.

I have a Victron Multiplus2 3000 48V, 50A discharge, 29A charge with 18* 3,2V 302Ah LiFePos and JBD-BMS and sb-ble branch

My (artificial) limits for the battery are:
MAX_BATTERY_CHARGE_CURRENT    = 100.0
MAX_BATTERY_DISCHARGE_CURRENT = 100.0

"Real" Limits would probably be 150A for 0,5C or 200A being the Limit of the BMS but as the Multiplus never comes near anything that would harm the batteries I guess it's negligible and only relevant for the values based on them.

Currently it charges bulk with 29A until the Battery reaches 99%, then it drops to 10A until the batteries reach 3,45A, then the Voltage drops to 60,24V, Parameter limits are 60,3V (3,35V * 18 FloatCell Voltage) and the Multiplus stays in "Absorption" for I guess ever. SOC 100%, Solar power available via AC1. 

Questions: 
* is this "normal" or should the Multiplus switch to Pass-Through or something else (it did before I changed the setup, but the old settings were a quick fix)?

* Does it make sense? My perspective is battery life & efficiency. I don't need the full Ah currently, so I think about lowering the voltages even further. 

My Config:

```
LINEAR_LIMITATION_ENABLE = True

MAX_BATTERY_CHARGE_CURRENT    = 100.0
MAX_BATTERY_DISCHARGE_CURRENT = 100.0


CVCM_ENABLE = True

MIN_CELL_VOLTAGE   = 3.00
MAX_CELL_VOLTAGE   = 3.45
FLOAT_CELL_VOLTAGE = 3.35


PENALTY_AT_CELL_VOLTAGE = 3.45, 3.55, 3.6
PENALTY_BATTERY_VOLTAGE = 0.01,  1.0, 2.0
PENALTY_RECALCULATE_EVERY = 60

MAX_VOLTAGE_TIME_SEC = 900
SOC_LEVEL_TO_RESET_VOLTAGE_LIMIT = 90




CCCM_CV_ENABLE = True
DCCM_CV_ENABLE = True

CELL_VOLTAGES_WHILE_CHARGING   = 3.55, 3.50, 3.45, 3.35
MAX_CHARGE_CURRENT_CV_FRACTION =    0, 0.05,  0.5,    1

CELL_VOLTAGES_WHILE_DISCHARGING   = 2.70, 2.80, 2.90, 3.10
MAX_DISCHARGE_CURRENT_CV_FRACTION =    0,  0.1,  0.5,    1


CCCM_T_ENABLE = True
DCCM_T_ENABLE = True
TEMPERATURE_LIMITS_WHILE_CHARGING = 0,   2,   5,  10,  15, 20, 35,  40, 55
MAX_CHARGE_CURRENT_T_FRACTION     = 0, 0.1, 0.2, 0.4, 0.8,  1,  1, 0.4,  0

TEMPERATURE_LIMITS_WHILE_DISCHARGING = -20,   0,   5,  10, 15, 45, 55
MAX_DISCHARGE_CURRENT_T_FRACTION     =   0, 0.2, 0.3, 0.4,  1,  1,  0


CCCM_SOC_ENABLE = True
DCCM_SOC_ENABLE = True

CC_SOC_LIMIT1 = 99
CC_SOC_LIMIT2 = 98
CC_SOC_LIMIT3 = 91

CC_CURRENT_LIMIT1_FRACTION = 0.1
CC_CURRENT_LIMIT2_FRACTION = 0.3
CC_CURRENT_LIMIT3_FRACTION = 0.5

DC_SOC_LIMIT1 = 10
DC_SOC_LIMIT2 = 20
DC_SOC_LIMIT3 = 30

DC_CURRENT_LIMIT1_FRACTION = 0.1
DC_CURRENT_LIMIT2_FRACTION = 0.3
DC_CURRENT_LIMIT3_FRACTION = 0.5


TIME_TO_SOC_POINTS = 100, 75, 50, 25, 0
TIME_TO_SOC_VALUE_TYPE = 2
TIME_TO_SOC_RECALCULATE_EVERY = 60
TIME_TO_SOC_INC_FROM = False


BMS_TYPE = LltJbd

PUBLISH_CONFIG_VALUES = 1

BATTERY_CELL_DATA_FORMAT = 1

MIDPOINT_ENABLE = False

SOC_LOW_WARNING = 5
SOC_LOW_ALARM   = 1

```

**Comment by mr-manuel** [comment, marked as answer]
The MultiPlus is showing you what you have configured with VE.Config. When you connect a BMS then the BMS takes over for the charging, if you enabled DVCC, what you shoudl. So you can ignore the Bulk/Absorption/Float state displayed from the MultiPlus.

The default values are already a good configuration, unless you have a very special setup or special needs.
